.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET = erwfwerfwerfwerasdfbsdafbsdafbf
```

Comment:
You want to put your .env in your .gitignore typically (so that people can't see your secret).